src/renderRepoCard.js

```diff
@@ -52,19 +52,23 @@ const renderRepoCard = (repo, options = {}) => {
         <text data-testid="lang" class="gray" x="15">${langName}</text>
       </g>
 
-      <g transform="translate(${155 - shiftText}, 100)">
-        <svg class="icon" y="-12" viewBox="0 0 16 16" version="1.1" width="16" height="16">
-          ${icons.star}
-        </svg>
-        <text data-testid="stargazers" class="gray" x="25">${totalStars}</text>
-      </g>
+      ${stargazers.totalCount > 0 && `
+        <g transform="translate(${155 - shiftText}, 100)">
+          <svg class="icon" y="-12" viewBox="0 0 16 16" version="1.1" width="16" height="16">
+            ${icons.star}
+          </svg>
+          <text data-testid="stargazers" class="gray" x="25">${totalStars}</text>
+        </g>
+      `}
 
-      <g transform="translate(${220 - shiftText}, 100)">
-        <svg class="icon" y="-12" viewBox="0 0 16 16" version="1.1" width="16" height="16">
-          ${icons.fork}
-        </svg>
-        <text data-testid="forkcount" class="gray" x="25">${totalForks}</text>
-      </g>
+      ${totalForks > 0 && `
```

Comment:
This looks good, but what if a repo has 0 stars and 1 fork? 

Then the design would have a gap in-between the `fork` and the `language` i think

Reply:
Right :)))).
I will fix it

Reply:
I think i need to implement some kind of `Flex System` to solve this common layout issues